From 3b14acdbcc39a43cdeac65fd9e49775f887fe5ef Mon Sep 17 00:00:00 2001
From: Robert Bragg <robert@linux.intel.com>
Date: Mon, 23 Sep 2013 17:28:30 +0000
Subject: remove spurious COGL_GTYPE_DEFINE_CLASS reference

Commit 7b25c8f5caad mistakenly added a reference to a
COGL_GTYPE_DEFINE_CLASS() macro that does not yet exist so this patch
removes it.

Reviewed-by: Neil Roberts <neil@linux.intel.com>
---
diff --git a/cogl/cogl-attribute-buffer.c b/cogl/cogl-attribute-buffer.c
index afa945b..20c093d 100644
--- cogl/cogl-attribute-buffer.c
+++ cogl/cogl-attribute-buffer.c
@@ -37,7 +37,6 @@
 static void _cogl_attribute_buffer_free (CoglAttributeBuffer *array);
 
 COGL_BUFFER_DEFINE (AttributeBuffer, attribute_buffer);
-COGL_GTYPE_DEFINE_CLASS (AttributeBuffer, attribute_buffer);
 
 CoglAttributeBuffer *
 cogl_attribute_buffer_new_with_size (CoglContext *context,
--
cgit v0.9.2
